sound/soc/intel/atom/sst/sst.c

```diff
@@ -49,7 +49,7 @@ static irqreturn_t intel_sst_interrupt_mrfld(int irq, void *context)
 	union ipc_header_mrfld header;
 	union sst_imr_reg_mrfld imr;
 	struct ipc_post *msg = NULL;
-	unsigned int size = 0;
+	unsigned int size;
```

Comment:
Would it be better to move ``unsigned int size;`` to line 92 inside ``if (header.p.header_high.part.large) {}``?

Reply:
we could in this case, but like I said in the answer to @lyakh using local scopes leads to shadowed variables. I don't feel one solution is better than the other...